woodwork/serialize.py

```diff
@@ -17,7 +17,7 @@
 dd = import_or_none('dask.dataframe')
 ks = import_or_none('databricks.koalas')
 
-SCHEMA_VERSION = '10.0.0'
+SCHEMA_VERSION = '10.1.0'
```

Comment:
Maybe we should bump the patch version here instead of the minor version? The current release of Woodwork will be able to read in any data serialized with this code as long as `Unknown` is not used. Similarly, this version of Woodwork will be able to read in any data serialized with the current release without issue.

Reply:
done!